client/src/redux/reducers/datafiles.reducers.js

```diff
@@ -1,10 +1,17 @@
 export const initialSystemState = {
-  defaultHost: '',
-  systemList: [],
-  error: false,
-  errorMessage: null,
-  loading: false,
-  definitions: []
+  datafiles: {
+    list: [],
```

Comment:
@jchuahtacc  is suggesting `systems` or `storageSystems`. gonna go with `storageSystems`

Reply:
@jchuahtacc 
there is also a list in definitions in this PR.  should i rename that too?  what about: 
```
{
  storage: {
    configuration: [],
    error: false,
    errorMessage: null,
    loading: false,
    defaultHost: ''
  },
  definitions: {
    systems: [],
    error: false,
    errorMessage: null,
    loading: true
  }
};

```

Reply:
@jchuahtacc  and I discussed on the phone.  I'm separating `systemsList` and `definitions` in this PR as we need to know there loading/error state separately to fix the bug.  but we'll really need to pull those out of the datafiles.reducer.js and put them in a storage reducer and a systems reducer.  This will be done in a follow up to not confuse the fix here [Adding ticket]. 

So for now we are going with
{
  storage: {
    configuration: [],
    error: false,
    errorMessage: null,
    loading: false,
    defaultHost: ''
  },
  definitions: {
    list: [],
    error: false,
    errorMessage: null,
    loading: true
  }
};